cmd/agent/app/integrations.go

```diff
@@ -51,11 +51,16 @@ var (
 	wheelPackageNameRe  = regexp.MustCompile("Name: (\\S+)")           // e.g. Name: datadog-postgres
 	versionSpecifiersRe = regexp.MustCompile("([><=!]{1,2})([0-9.]*)") // Matches version specifiers defined in https://packaging.python.org/specifications/core-metadata/#requires-dist-multiple-use
 
-	allowRoot    bool
-	verbose      int
-	useSysPython bool
-	versionOnly  bool
-	localWheel   bool
+	allowRoot           bool
+	verbose             int
+	useSysPython        bool
+	versionOnly         bool
+	localWheel          bool
+	rootDir             string
+	pythonMajorVersion  string
```

Comment:
These should be overridable by flags. Say you want to install an integration on the python3 environment but are currently still running the agent on Python 2.7. We should give careful consideration to the other options as well.